app/liquid_tags/codesandbox_tag.rb

```diff
@@ -14,18 +15,39 @@ def render(_context)
   private
 
   def parse_id(input)
-    input_no_space = input.delete(" ")
-    if valid_id?(input_no_space)
-      input_no_space
-    else
-      raise StandardError, "Invalid codesandbox Id"
+    id = input.split(" ").first
+    raise StandardError, "Invalid codesandbox ID" unless valid_id?(id)
```

Comment:
Might as well capitalize CodeSandbox while we're here:
```suggestion
    raise StandardError, "Invalid CodeSandbox ID" unless valid_id?(id)
```